linalg/simd/auto.hpp

```diff
@@ -117,6 +117,11 @@ struct MFEM_AUTOSIMD_ALIGN__ AutoSIMD
       return r;
    }
 
+   inline MFEM_ALWAYS_INLINE AutoSIMD operator+() const
+   {
+      return *this;
+   }
```

Comment:
I think, this method should also be added to the other files in this directory -- they add specialization for this class for specific architectures. To try this on a supported architecture, e.g. `x86`, enable `MFEM_USE_SIMD=YES`.

Reply:
I added them. Is there a way to test that all of these are working properly?

Reply:
You can enable `MFEM_USE_SIMD=YES` and then set the architecture option in the makefile in `miniapps/performance`, e.g. for GCC here: https://github.com/mfem/mfem/blob/d09ab725b55cf86455d42b4611dccd1a835242ed/miniapps/performance/makefile#L93

For GCC and clang with `x86`, you can try replacing `-march=native` with: `-march=X` where `X` is `core2`, `skylake`, `skylake-avx512` to use respectively `m128.hpp`, `m256.hpp`, `m512.hpp`. The example prints the selected SIMD width here: https://github.com/mfem/mfem/blob/d09ab725b55cf86455d42b4611dccd1a835242ed/miniapps/performance/ex1.cpp#L127-L128

Reply:
Ok, thanks. It appears to be working for the ones I was able to test.